test/Collector.test.ts

```diff
@@ -0,0 +1,185 @@
+import { expect, use } from 'chai';
+import Collector from '../build/contracts/Collector.json';
+import TestToken from '../build/contracts/TestToken.json';
+import {
+    deployContract,
+    loadFixture,
+    MockProvider,
+    solidity
+} from 'ethereum-waffle';
+import { BaseContract } from '@ethersproject/contracts';
+import { Wallet } from '@ethersproject/wallet';
+
+use(solidity);
+
+const buildPartners = (
+    wallets: Wallet[],
+    share1?: number,
+    share2?: number,
+    share3?: number,
+    share4?: number
+) => {
+    const [owner, remainder, partner1, partner2, partner3, partner4] = wallets;
```

Comment:
I wonder if this line can be replaced by:
const [, , partner1, partner2, partner3, partner4] = wallets;

Reply:
agreed